base/ltluatex.dtx

```diff
@@ -1399,12 +1399,14 @@ local callbacktypes = callbacktypes or {
 % \changes{v1.1j}{2019/06/18}{process\_pdf\_image\_content added}
 % \changes{v1.1j}{2019/10/22}{page\_objnum\_provider and process\_pdf\_image\_content classified data}
 % \changes{v1.1l}{2020/02/02}{page\_order\_index added}
+% \changes{v1.1t}{2021/04/18}{input\_level\_string added}
 %    \begin{macrocode}
   finish_pdffile            = data,
   finish_pdfpage            = data,
   page_objnum_provider      = data,
   page_order_index          = data,
   process_pdf_image_content = data,
+  input_level_string        = data,
```

Comment:
s/data/exclusive/

The `data` type is for callbacks which take data as their first argument, potentially modify it a little and return it again. But ´input_level_string`takes a completely different data type as input and output.

Reply:
Thanks, I wondered which would be the right type here